manifests/init.pp

```diff
@@ -4,46 +4,20 @@
 #
 # === Parameters:
 #
-# $enable_ostree::                      Enable ostree content plugin, this requires an ostree install
-#
 # $enable_yum::                         Enable rpm content plugin, including syncing of yum content
 #
 # $enable_file::                        Enable file content plugin
 #
-# $enable_puppet::                      Enable puppet content plugin
-#
 # $enable_docker::                      Enable docker content plugin
```

Comment:
This parameter is now unused. https://github.com/theforeman/puppet-foreman_proxy_content/pull/311 would use it again. Not sure what the order should be, but I do think it makes sense have a parameter to turn it on/off.

Reply:
I was expecting https://github.com/theforeman/puppet-foreman_proxy_content/pull/311 to go in and I would update on to it.

Reply:
Sounds good to me.